### PR TITLE
add: `doas-git`

### LINKED ISSUE #591: PacReq: `doas-git`
doas is an alternative to sudo, that aims to provide 95% of sudo's functionality while having a substantially smaller codebase. It was originally created for the OpenBSD project, but has been ported to Linux.

URL: https://github.com/slicer69/doas
Building Guide: https://github.com/slicer69/doas#installing-build-tools

### PR DESCRIPTION
**Pull request by wizard-28**
## Progress
- [x] Edit `packagelist`
- [x] Add initial pacscript
- [x] Contact devs: https://github.com/slicer69/doas/issues/93
- [x] Add `maintainer` to pacscript: Resolved to @wizard-28 (https://github.com/slicer69/doas/issues/93#issuecomment-1003067280)

Closes #591

### CODE REVIEW

**Comment by wizard-28** [comment]
@marzeq, the script is done, but before this gets merged we need to wait for a response from slicer69. If we don't get a response before 1 week, then we'll merge this.

**Review by Henryws — APPROVED** [review]

**Pull request merged**
